runtime/hydradx/src/evm/precompiles/erc20_mapping.rs

```diff
@@ -22,15 +22,20 @@
 use crate::evm::EvmAddress;
 use crate::Runtime;
 use hex_literal::hex;
-use hydradx_traits::{evm::Erc20Mapping, RegisterAssetHook};
+use hydradx_traits::{evm::Erc20Encoding, RegisterAssetHook};
 use primitive_types::{H160, H256};
 use primitives::AssetId;
 
 pub struct HydraErc20Mapping;
 
-/// Erc20Mapping logic for HydraDX
+// impl<T: pallet_asset_registry::Config + BoundErc20> Erc20Mapping<AssetId> for HydraErc20Mapping {
```

Comment:
we can remove this if not needed

Reply:
i forgot to finish implementation of it :)